### ISSUE TITLE
Adguard website opens up every time I open my browser

### ISSUE THREAD

**Issue by Alex-302** [AdguardTeam/AdguardBrowserExtension#2189]
@adguard-bot commented on [Thu Sep 15 2022](https://github.com/AdguardTeam/AdguardFilters/issues/129812)

### Issue URL (Annoyance)
 [https://welcome.adguard.com/v3/thankyou.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwelcome.adguard.com%2Fv3%2Fthankyou.html)
### Comment
Username: @Magmalight
> Adguard website opens up every time I open my browser and it's kind of annoying. Also, the settings are always turned off/disabled/unchecked when opening the browser and don't seem to be saved once I change them to how I want them to be. How can I fix this?

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  ![Screenshot 1](https://reports-img.adguard.com/tSBHcJw.png)
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v4.1.22
Browser: | Firefox
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection
Other extensions used: | uBlock Origin, AdBlocker Ultimate, Webmail Ad Blocker

**Comment by Alex-302** [comment]
@Magmalight

**Comment by maximtop** [comment]
@Magmalight, do you have any apps for clearing storage?

**Comment by Birbber** [comment]
@Magmalight Any news? Could you check if there are any apps that might clear the storage?

**Comment by xFlickerx** [comment]
I have the same problem. Firefox, use only AdGuard Browser extension v4.1.22, extension standard settings. Every time I open my browser and every time AdGuard website opens up... No storage cleaners, no "PC Boosters", nothing.

**Comment by maximtop** [comment]
> I have the same problem. Firefox, use only AdGuard Browser extension v4.1.22, extension standard settings. Every time I open my browser and every time AdGuard website opens up... No storage cleaners, no "PC Boosters", nothing.

@xFlickerx
Does the browser has default settings related to cache or cleaning?
Can we arrange an online session via anydesk? You can write me in the telegram - maximtop

**Comment by TOLIE-OFFICIAL** [comment]
i have the same problem. 
I am not sure if it`s because i am a Chinese., so Firefox blocked us.  or it`s adgard itself

**Comment by TOLIE-OFFICIAL** [comment]
it  not only open Adguard website, but also will clean all your settings.

**Comment by scrutinizer11** [comment]
I second the OP. After updating to 4.1.36, the extension opens its own tab with general settings that are always off upon launch regardless of the status. I liked how it was before more. Firefox 78ESR. Make it "update filters" in the background.

**Comment by maximtop** [comment]
>After updating to 4.1.36, the extension opens its own tab with general settings that are always off upon launch regardless of the status set previously
how does it look? are there any errors on the background page?

> Make it "update filters" in the background.
what do you mean? can you provide screenshots or a video?

**Comment by scrutinizer11** [comment]
> > After updating to 4.1.36, the extension opens its own tab with general settings that are always off upon launch regardless of the status set previously
> 
> how does it look? are there any errors on the background page?
> 
> > Make it "update filters" in the background.
> 
> what do you mean? can you provide screenshots or a video?

[https://streamable.com/8nqy6v](url)

**Comment by scrutinizer11** [comment]
Where can I download 4.1.22 for Firefox?

**Comment by maximtop** [comment]
you can download it here
https://github.com/AdguardTeam/AdguardBrowserExtension/releases/tag/v4.1.22

Maybe the next week you can help us to detect this bug? can you write me an email at maximtop@gmail.com?

**Comment by scrutinizer11** [comment]
Ok, I had found that on my own. 4.1.22 doesn't have this problem.

**Comment by scrutinizer11** [comment]
> Maybe the next week you can help us to detect this bug? can you write me an email at [maximtop@gmail.com](mailto:maximtop@gmail.com)?

What info do you want me to enclose?

**Comment by maximtop** [comment]
I need your help to debug this issue on your computer via anydesk if it is possible.

**Comment by Birbber** [comment]
@scrutinizer11 Is this issue still reproducible? Have you been able to debug it along with @maximtop?

**Comment by scrutinizer11** [comment]
No, I couldn't find a free slot in my schedule. How do we do that?

**Comment by Birbber** [comment]
We're gonna try to debug the issue on our end. Will let you know once we get any information.

**Comment by johnjoemorgan** [comment]
This is happening again on Firefox 109.0.0 & 109.0.1 (64-bit)

**Comment by QueerestPunk** [comment]
This issue has been ongoing for me since October 2022 (posted about it here: https://github.com/AdguardTeam/AdguardBrowserExtension/issues/2207). I eventually just disabled Adguard because it was so annoying to deal with. I hope a fix is found at some point, as this was a really great extension before it stopped working

**Comment by maximtop** [comment]
Sometimes settings file might become corrupted. For example, there was a user with a similar problem. 
Here he wrote what helped him on Ubuntu.

```
I found out that I can open the AdGuard settings and get the UUID from the URI there. Then I did a system-wide search and found:

/home/fabian/.mozilla/firefox/rsbg1oxz.default-release-1/storage/default/moz-extension+++988a9de9-a067-48ca-ac71-aa876ee198da

/home/fabian/.mozilla/firefox/rsbg1oxz.default-release-1/storage/default/moz-extension+++988a9de9-a067-48ca-ac71-aa876ee198da^userContextId=4294967295

That contains a bunch of cryptically named files. I could have figured out which one was corrupt, but it was faster to just copy my custom rules, delete both folders, get one final welcome screen, set all settings again and paste my custom rules.

Now the issue is fixed, thanks!
```
